app/src/main/res/values-ar/about.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_close">Close</string>
```

Comment:
This and the following two strings can be translated

Reply:
Done